packages/calcite-components/src/components/checkbox/checkbox.scss

```diff
@@ -4,16 +4,31 @@
  * These properties can be overridden using the component's tag as selector.
  *
  * @prop --calcite-checkbox-size: Specifies the component's height and width.
+ * @prop --calcite-checkbox-box-shadow: Specifies the component's svg box color.
```

Comment:
I should've spotted this sooner, but can you revisit the prop description to not mention implementation details (i.e., svg)? Applies to new props.